### PR TITLE
add daemonset to refresh springtrader cache images

### PR DESCRIPTION
**Pull request by ChrisSchreiber**
Not 100% sure this is the best place for this. If anyone has a better idea let me know.

### CODE REVIEW

File: .cache/cache-refresh-daemonset.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    spec:
+      containers:
+      - name: springtrader-cache-refresh
+        image: artifactory.toolchain.lead.prod.liatr.io/docker-registry/flywheel/builder-image-skaffold:v1.1.1-1-g6b3c49e
+        command: ["/bin/sh"]
+        args: ["-c", "while true; do docker pull artifactory.toolchain.lead.prod.liatr.io/docker-registry/liatrio/springtrader-builder:latest; docker pull artifactory.toolchain.lead.prod.liatr.io/docker-registry/liatrio/springtrader-runner:latest; sleep 600; done"]
+        volumeMounts:
+        - mountPath: /var/run/docker.sock
+          name: docker
+      volumes:
```
Comment (mrparkers) [inline .cache/cache-refresh-daemonset.yaml:32]: I think `/var/lib/docker` needs to be mounted too.

File: .cache/cache-refresh-daemonset.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: springtrader-cache-refresh
```
Comment (mrparkers) [inline .cache/cache-refresh-daemonset.yaml:4]: default namespace?

File: .cache/cache-refresh-daemonset.yaml (hunk 1 above)
Comment (ChrisSchreiber) [reply .cache/cache-refresh-daemonset.yaml:32]: I used the same mounts that the jenkins agent containers use (which do not include /var/lib/docker). My guess is that all of the container interactions go through the daemon and do not access the docker filesystem directly so it's not needed but I could be wrong. Do you know for sure they need the /var/lib/docker?

Comment (mrparkers) [reply .cache/cache-refresh-daemonset.yaml:32]: No, it was just a guess.  If it's working then its probably not needed

**Review by mrparkers — APPROVED** [review]

**Pull request merged**